diff --git a/node_modules/@nuxtjs/tailwindcss/dist/module.mjs b/node_modules/@nuxtjs/tailwindcss/dist/module.mjs
index ba6a736..b983a4b 100644
--- a/node_modules/@nuxtjs/tailwindcss/dist/module.mjs
+++ b/node_modules/@nuxtjs/tailwindcss/dist/module.mjs
@@ -83,7 +83,7 @@ const module = defineNuxtModule({
     tailwindConfig._hash = String(Date.now());
     const postcssOptions = nuxt.options.postcss || nuxt.options.build.postcss.postcssOptions || nuxt.options.build.postcss;
     postcssOptions.plugins = postcssOptions.plugins || {};
-    postcssOptions.plugins["tailwindcss/nesting"] = postcssOptions.plugins["tailwindcss/nesting"] ?? {};
+    postcssOptions.plugins["@tailwindcss/nesting"] = postcssOptions.plugins["@tailwindcss/nesting"] ?? {};
     postcssOptions.plugins["postcss-custom-properties"] = postcssOptions.plugins["postcss-custom-properties"] ?? {};
     postcssOptions.plugins.tailwindcss = tailwindConfig;
     if (isNuxt2()) {
